src/editor/nodeDefs/io/arrays.js

```diff
@@ -0,0 +1,205 @@
+                $("#node-form-row-param2Name").html(functionDef.params[0].name);
+                $("#node-form-row-param2Type").html(functionDef.params[0].type);
+                $("#node-form-row-param3Name").html(functionDef.params[1].name);
+                $("#node-form-row-param3Type").html(functionDef.params[1].type);
+                $("#node-form-row-description").html(functionDef.description);
+                $("#node-form-row-param2Name").show();
+                $("#node-form-row-param2Type").show();
+                $("#node-form-row-param3Name").show();
+                $("#node-form-row-param3Type").show();
+                $("#node-form-row-description").show();
+                $("#node-form-row-param2Row").show();
+                $("#node-form-row-param3Row").show();
+              } else {
+                $("#node-form-row-param2Name").html(functionDef.params[0].name);
+                $("#node-form-row-param2Type").html(functionDef.params[0].type);
+                $("#node-form-row-param3Name").html(functionDef.params[1].name);
+                $("#node-form-row-param3Type").html(functionDef.params[1].type);
+                $("#node-form-row-param4Name").html(functionDef.params[2].name);
+                $("#node-form-row-param4Type").html(functionDef.params[2].type);
+                $("#node-form-row-description").html(functionDef.description);
+                $("#node-form-row-param2Name").show();
+                $("#node-form-row-param2Type").show();
+                $("#node-form-row-param3Name").show();
+                $("#node-form-row-param3Type").show();
+                $("#node-form-row-param4Name").show();
+                $("#node-form-row-param4Type").show();
+                $("#node-form-row-description").show();
+                $("#node-form-row-param2Row").show();
+                $("#node-form-row-param3Row").show();
+                $("#node-form-row-param4Row").show();
+              }
+            }
+          }
+
+          handleFunc(myFuncDef);
+
+          var funcInput = $("#node-input-func");
+          funcInput.change(function (){
+            console.log('funcInput changed', this.value);
+            handleFunc(arrayFunctions[this.value]);
+          })
+        },
+
+        render: function (){
+          return (
+            <div>
+              <div className="form-row">
+                <i className="fa fa-hand-paper-o" /> Handling
+                  <span style={{marginLeft: "5.5%"}}>Parse <code>msg.payload</code> to JSON? </span>
+                  <input type="checkbox" id="node-input-wantsPayloadParsed"style={{width: "inherit"}}></input>
+              </div>
+
+              <div className="form-row">
+                <i className="fa fa-gears"></i><label htmlFor="node-input-func" style={{marginLeft:"1%"}}>Func</label>
+                <select type="text" id="node-input-func" style={{width: "74%", marginLeft: "-0.8%"}}>
```

Comment:
In the future we should turn this option in to a JS array and then use .map go over and render a list.